src/js/components/covid19/AmountsVisualization.jsx

```diff
@@ -66,6 +75,15 @@ const AmountsVisualization = ({
     const [obligationLineDataTwoOverlap, setObligationLineDataTwoOverlap] = useState(false);
     const [drawRemainingBalanceItems, setDrawRemainingBalanceItems] = useState(true);
     const [drawRemainingBalanceText, setDrawRemainingBalanceText] = useState(true);
+    const [showTotalTooltip, setShowTotalTooltip] = useState(false);
```

Comment:
Recommend condensing this into a single method where activeTooltip is an enum; one of string "tooltipname". From there we can just render a single  `TooltipWrapper` component and use datamapping to invoke `showTooltip` w/ the proper tooltip name for each `onFocus` handler. This is a massive component and I think this change would make sense if only to limit the number of methods.